### PR TITLE
Richer atom feeds

### PR DESCRIPTION
**Pull request by longhotsummer**
* add additional custom elements
* generate  a big honking all.atom feed.

### CODE REVIEW

**Comment by longhotsummer** [comment]
@jbothma

File: gazettes/archive.py [hunk 1]
```diff
@@ -110,40 +111,42 @@ def write_index():
...
-        entry.summary('%s is available for download at <a href="%s">%s</a>' % (
-            archived_gazette.full_title, archived_gazette.archive_url, archived_gazette.archive_url))
+    # everything
+    gazettes = session\
+        .query(ArchivedGazette)\
+        .order_by(desc(ArchivedGazette.created_at))\
+        .all()
+    write_feed(
+        gazettes,
+        title='South African Gazettes - Recently Added',
```
Comment (jbothma) [inline gazettes/archive.py]: not recently added

File: gazettes/archive.py [hunk 2]
```diff
@@ -110,40 +111,42 @@ def write_index():
...
+        description='Recently added South African Gazettes',
```
Comment (jbothma) [inline gazettes/archive.py]: ditto

**Comment by jbothma** [comment]
cool stuff. Couple of small issues above, otherwise nice!

**Comment by longhotsummer** [comment]
Shot. Updated and merging.

**Pull request merged**